[remark-wiki-link][m]: Add image size adjustment in `remark-wiki-link`

mohamedsalem401 opened the pull request:
For https://github.com/datopian/flowershow/issues/592

We add `width` and `height` to `img` tags, and also we add inline styling for the width and the height of the image.

olayway reviewed (changes requested):
Great. Just some minor changes for improved readability and efficiency, and an additional test for when only one dimension is specified.

Also, next time, please make sure to reduce noise by not including any redundant formatting changes, as this is extremely hard to read.

olayway commented:
@mohamedsalem401 Please remember to add a changeset file :)

rufuspollock commented:
@mohamedsalem401 can you do quick tweaks so we can get this merged.

mohamedsalem401 commented:
@rufuspollock @olayway 
I updated the PR.
- Added test case for single dimension `[[image.png|100]]`
- Applied the refactoring suggestions
- Updated tests
- Added changeset

All tests are running correctly.
